<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
                background-color: #2C2F33;
            }
            QLabel#label {
                color: #FFFFFF;
                font-family: 'Arial';
                font-size: 28px;
                font-weight: bold;
                background-color: transparent;
                padding: 10px;
            }
            QLabel#label_2, QLabel#label_3 {
                color: #7289DA;
                font-family: 'Arial';
                font-size: 20px;
                font-weight: normal;
                background-color: transparent;
            }
            QComboBox {
                background-color: #23272A;
                color: #FFFFFF;
                font-family: 'Arial';
                font-size: 16px;
                border-radius: 10px;
                padding: 10px;
                border: 1px solid #7289DA;
            }
            QComboBox:hover {
                border: 1px solid #677BC4;
            }
            QComboBox::drop-down {
                border: none;
                width: 20px;
            }
            QComboBox::down-arrow {
                image: url(:/icons/down_arrow.png);
                width: 12px;
                height: 12px;
            }
            QComboBox QAbstractItemView {
                background-color: #23272A;
                color: #FFFFFF;
                selection-background-color: #7289DA;
                selection-color: #FFFFFF;
                border: 1px solid #7289DA;
                border-radius: 5px;
            }
            QLineEdit {
                background-color: #23272A;
                color: #FFFFFF;
                font-family: 'Arial';
                font-size: 20px;
                border-radius: 10px;
                padding: 10px;
                border: 1px solid #7289DA;
            }
            QLineEdit:hover, QLineEdit:focus {
                border: 1px solid #677BC4;
            }
            QPushButton {
                background-color: #7289DA;
                color: #FFFFFF;
                font-family: 'Arial';
                font-size: 16px;
                font-weight: bold;
                border-radius: 10px;
                padding: 10px;
                border: none;
            }
            QPushButton:hover {
                background-color: #677BC4;
            }
            QPushButton:pressed {
                background-color: #5B6EAE;
            }
            QLCDNumber {
                background-color: #23272A;
                color: #43B581;
                border-radius: 10px;
                border: 1px solid #7289DA;
                padding: 10px;
            }
            QStatusBar {
                background-color: #23272A;
                color: #FFFFFF;
            }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Currency Converter Dynamic</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDown1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDown2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdpanel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
